scipy/stats/tests/test_distributions.py

```diff
@@ -3084,7 +3084,7 @@ def test_fit(self):
         assert_almost_equal(scale2, .02503, 4)
         assert_almost_equal(loc2, .03354, 4)
 
-    @pytest.mark.slow
+    @pytest.mark.xslow
```

Comment:
This test was changed in the last few days to be parametrized, with only a subset marked as `slow`. Let's look at the new timings in CI before marking it as `xslow`. I'll resolve the merge conflict now.

Reply:
Relevant lines in PR: https://github.com/scipy/scipy/pull/9523/files#diff-b1029c91884ad11273b8755da7282076d4e6f533d299cab11fef54744af7d070R3247-R3268

Reply:
Yes, see https://github.com/scipy/scipy/pull/15234#issuecomment-997042756.

Reply:
So do you want to put the `xslow` back now? The test has at least disappeared out of the top 10 after the rewrite. Maybe we should increase to `--durations=25` in CI?

Reply:
No, it's fine with me as-is if it's not showing up in the list. 
Instead of `--durations=25`, maybe something like `--durations-min=1.0` to see all tests longer than some threshold?